app/assets/javascripts/discourse/app/components/anonymous-topic-footer-buttons.js

```diff
@@ -0,0 +1,21 @@
+import Component from "@ember/component";
```

Comment:
Let's use a glimmer component for this as the old `@ember/component` is not recommended as the default going forward.

Reply:
I tried, but I couldn't find a way to re-use the `getTopicFooterButtons` exported function with a Glimmer component – if you have any suggestion.